tests/test_commands.py

```
@@ -1,7 +1,6 @@
-import filecmp
-
+import pandas as pd
 from click.testing import CliRunner
-
+import pyarrow as pa
```

Ruff (0.8.0)

`pyarrow` imported but unused

Remove unused import: `pyarrow`

(F401)